docs/source/software/systemwide/matlab.md

```diff
@@ -61,95 +61,372 @@ Use the following procedure if you need to install a MATLAB toolbox:
 1. You can now use the toolbox within MATLAB. When you are done, type `quit`.
 
 ## Using MATLAB Parallel Server
-
-The cluster has MATLAB Parallel Server installed. This section details an example of how you can set up and use the MATLAB Parallel Computing Toolbox. This walkthrough uses MATLAB 2023a launched as an interactive app on the Open OnDemand web portal. There are several parts to this walkthrough. We suggest that you read it through completely before starting. The parameters presented represent only one scenario.
-
-This walkthrough will use Open OnDemand, the web portal on the cluster, to launch MATLAB. You will then use the default cluster profile.This allows you to utilize the resources that are allocated to your Open On Demand MATLAB job.
-
-To start MATLAB and work with the Parallel Server, do the following:
+### Configuration of MATLAB client on the HPC
+The cluster has MATLAB Parallel Server installed and this section details an example of how you can set up and use the MATLAB Parallel Computing Toolbox on the HPC. This walkthrough uses MATLAB launched as an interactive app on the [Open OnDemand](ood.discovery.neu.edu) web portal. There are several parts to this walkthrough so we suggest that you read it through completely before starting. The parameters presented represent only one scenario and the results may differ when you run the examples.
```

Comment:
I would add a section specifying the MATLAB version and Parallel Computing Toolbox version, Some features may vary in different versions.

Reply:
Thank you and I have included the MATLAB version. Thank you!!